src/main/java/com/rk/rentacarapp/services/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.rk.rentacarapp.services.impl;
+
+import com.rk.rentacarapp.model.User;
+import com.rk.rentacarapp.repository.UserRepository;
+import com.rk.rentacarapp.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    UserRepository ur;
+
+    @Override
+    public User findByEmail(String email) {
+        return ur.findByEmail(email);
+    }
+
+    @Override
+    public void registerUser(User user) {
+        ur.save(user);
+    }
+
+    @Override
+    public List<User> findAllUsers() {
```

Comment:
I'd use get instead of find as a much more used term for things like this. Controller will be calling this when a GET request occurs